Chapter_2/dockerfiles/cpu-jupyter-notebook/README.md

```diff
@@ -0,0 +1,36 @@
+@Lenin please update this read me
```

Comment:
Please update this read me.

Also, can you explain each docker file on each read me? (what are the differences?)

Reply:
same as the comment in the above one.